src/main/java/frc/robot/systems/DriveFSMSystem.java

```diff
@@ -0,0 +1,233 @@
+package frc.robot.systems;
+
+// WPILib Imports
+import edu.wpi.first.wpilibj.SPI;
+
+// Third party Hardware Imports
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import com.revrobotics.ControlType;
+import com.kauailabs.navx.frc.AHRS;
+
+// Robot Imports
+import frc.robot.TeleopInput;
+import frc.robot.HardwareMap;
+
+public class DriveFSMSystem {
+	/* ======================== Constants ======================== */
+	public static final double WHEEL_DIAMETER_INCHES = 7.65;
+	public static final double kP_move_straight = 0.1;
```

Comment:
All constants should have all caps names, probably checkstyle is warning on this too